build.proj

```diff
@@ -1,51 +1,10 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="12.0" DefaultTargets="Build"
-  xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+<Project Sdk="Microsoft.Build.NoTargets">
+  <!-- This file should be removed once we move to Arcade -->
```

Comment:
Thanks for adding the comment!

Reply:
I will be adding `[ARCADE_REMOVE]` as a comment string in places we need to address post transition.